src/main/java/com/commercetools/sync/inventories/utils/InventoryUpdateActionUtils.java

```diff
@@ -0,0 +1,112 @@
+package com.commercetools.sync.inventories.utils;
+
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+import io.sphere.sdk.inventory.commands.updateactions.ChangeQuantity;
+import io.sphere.sdk.inventory.commands.updateactions.SetExpectedDelivery;
+import io.sphere.sdk.inventory.commands.updateactions.SetRestockableInDays;
+import io.sphere.sdk.inventory.commands.updateactions.SetSupplyChannel;
+import io.sphere.sdk.models.Reference;
+
+import javax.annotation.Nonnull;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+import static com.commercetools.sync.commons.utils.CommonTypeUpdateActionUtils.buildUpdateAction;
+
+/**
+ * This class provides static utility methods for building update actions related to inventories.
+ */
+public final class InventoryUpdateActionUtils {
+
+    private InventoryUpdateActionUtils() {
+        throw new AssertionError();
```

Comment:
Ok, so
1) class won't be subclassed
2) constructor is is not invoked anywhere inside the class
3) is it going to be used with reflection?

Example with Objects is much broader as it is used in almost every Java project.
So in that case such hard precautions are justified but here I don't see any reason.